src/main/java/io/ebeaninternal/server/persist/dml/DmlBeanPersister.java

```diff
@@ -85,7 +85,7 @@ private int execute(PersistRequestBean<?> request, PersistHandler handler) {
       throw new PersistenceException(msg, e);
 
     } finally {
-      if (!batched && handler != null) {
+      if (!batched) {
```

Comment:
The handler variable is accessed several times before this point. If the variable was ever null, it would have thrown before this point. Therefore, a nullcheck is pointless. Feel free to check the rest of the code for this, it's not used anywhere else but in this class.